Resources/Prototypes/Nyanotrasen/Entities/Clothing/psionic_clothing.yml

```diff
@@ -64,3 +64,18 @@
   - type: ClothingGrantPsionicPower
     power: NoosphericZapPower
   - type: Psionic
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckRecoveryConePsionic
+  name: radar dish
+  description: yo, is shark week on yet?
```

Comment:
It's not a radar dish, and the description is inscrutable to me.
Just comment this entire item out with a note that it can have the RF sensitivity power when that is fixed _(which requires all of ChatSystem to be refactored, but...)_